app/src/test/java/org/simple/clinic/security/pin/PinEntryCardControllerTest.kt

```diff
@@ -68,75 +65,83 @@ class  PinEntryCardControllerTest {
 
   @Test
   fun `when 4 digits are entered then the PIN should be submitted automatically`() {
+    val pinDigest = "1234"
     uiEvents.onNext(PinTextChanged("1"))
     uiEvents.onNext(PinTextChanged("12"))
     uiEvents.onNext(PinTextChanged("123"))
     uiEvents.onNext(PinTextChanged("1234"))
+    uiEvents.onNext(PinDigestToVerify(pinDigest))
 
-    verify(passwordHasher, times(1)).compare(loggedInUser.pinDigest, "1234")
+    verify(passwordHasher, times(1)).compare(pinDigest, "1234")
   }
 
   @Test
   fun `when the PIN is submitted then it should be validated`() {
+    val pinDigest = "1234"
     uiEvents.onNext(PinTextChanged("1234"))
+    uiEvents.onNext(PinDigestToVerify(pinDigest))
 
-    verify(passwordHasher).compare(loggedInUser.pinDigest, "1234")
+    verify(passwordHasher).compare(pinDigest, "1234")
```

Comment:
Similarly here, why do we have the variable `pinDigest`?

Reply:
Same as the above comment.